app/assets/stylesheets/restaurant.scss

```diff
@@ -11,5 +11,9 @@ header {
 }
 
 .links:hover {
-  cursor: url('cursorburger.png'), default;
+  cursor: image-url('cursorburger.png'), default;
+}
+
+.login {
+  text-align: right;
 }
```

Comment:
If you are using Foundation, this can also be accomplished with their pre-existing `.text-right` helper class. http://foundation.zurb.com/sites/docs/typography-helpers.html#text-alignment